src/Doctrine/DoctrineHelper.php

```diff
@@ -271,6 +272,19 @@ public static function getPropertyTypeForColumn(string $columnType): ?string
             'ulid' => '\\'.Ulid::class,
             default => null,
         };
+
+        if (null === $propertyType && Type::getTypeRegistry()->has($columnType)) {
+            $reflection = new \ReflectionClass(\get_class(Type::getTypeRegistry()->get($columnType)));
+            if ($reflection->hasMethod('convertToPHPValue') && $returnType = $reflection->getMethod('convertToPHPValue')->getReturnType()) {
+                if ($returnType->isBuiltin()) {
+                    $propertyType = $returnType->getName();
+                } else {
+                    $propertyType = '\\'.$returnType->getName();
+                }
```

Comment:
We could do something like `return $returnType->isBuitin() ? $returnType->getName() : ......`